packages/smooth_app/lib/pages/user_preferences_page.dart

```diff
@@ -34,10 +37,49 @@ class UserPreferencesPage extends StatelessWidget {
         _reorderGroups(productPreferences.attributeGroups!);
     final List<String> orderedImportantAttributeIds =
         productPreferences.getOrderedImportantAttributeIds();
+
     return Scaffold(
       appBar: AppBar(
         title: Text(appLocalizations.myPreferences),
         actions: <Widget>[
+          //Temporary for checking the credentials
```

Comment:
I don't know what we're going to see in the end: all the widgets are commented as temporary or test.